README.md

```diff
@@ -23,3 +23,18 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+## Answer:
+
+We know that when it comes to finding the closest nodes in this code and finding the closest unvisited node and finding its distance will be $O|V|$ time. It is also executed V times leading to us getting $O|V^2|$ for the time in finding the closest edge. 
+
+Our code also has to update the distances once they have been found by the neighboring node. This time would be $O(E)$. E is the edges in the code. 
+
+The time complexity for the vertices would be $O|V^2|$. Everything combined it would be $O((V^2)+E)$ or $O((|E| + |V|)log|V|)$
```

Comment:
Wait, where does the log come from suddenly?

Reply:
Sorry I was following the slides. I messed up on that because I skipped the slide and did the part with the heaps and sparse.